docs/en/workbench/how_to/create_workbench.mdx

```diff
@@ -0,0 +1,60 @@
+---
+weight: 14
+---
+
+# Create Workbench
+
+## Prerequisites
+
+- Ensure you have `kubectl` configured and connected to your cluster.
+- Ensure you have created `WorkspaceKind` (cluster resource).
+
+### Steps
+
+<Steps>
+### Create PVC
+  1. Login, go to the **Alauda Container Platform** page.
+  2. Click **Storage** > **PersistentVolumeClaims** to enter the **PVC** list page.
+  3. Find the **Create PVC** button, click **Create**, and enter the info.
+
+### Create SA
+  `kubectl` apply the example yaml as blow.
```

Comment:
Create a file `workbench-sa-with-rolebinding.yaml` with following lines, and run `kubectl apply -f workbench-sa-with-rolebinding.yaml` to create service account used by workbench instances in the namespace.

Reply:
Maybe these lines should be moved to `create_workspace_kind.mdx` file?